estate/models/offer.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+class offer(models.Model):
+    _name = 'estate.house_offer'
+
+    price = fields.Float()
+    status = fields.Selection(selection=[('Accepted','Accepted'),('Refused','Refused')], copy=False)
```

Comment:
Use lowercase keys for selection values, as they are stored in the database and are usually represented in lowercase in Odoo.
```suggestion
    status = fields.Selection(selection=[('accepted','Accepted'),('refused','Refused')], copy=False)
```